src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php

```diff
@@ -32,10 +29,10 @@ function let()
                 'driver' => 'doctrine/orm',
                 'templates' => 'SyliusProductBundle:Product',
                 'classes' => [
-                    'model' => Product::class,
+                    'model' => 'Sylius\Component\Product\Model\Product',
```

Comment:
IMHO real example is much more readable. Should it be changed? ping @michalmarcinkowski @pjedrzejewski